handlers/schemas/UploadVOD.json

```diff
@@ -0,0 +1,66 @@
+{
+    "type": "object",
+    "properties": {
+        "url": {
+            "type": "string",
+            "format": "uri"
+        },
+        "callback_url": {
+            "type": "string",
+            "format": "uri"
+        },
+        "mp4_output": {
+            "type": "boolean"
+        },
+        "output_locations": {
+            "type": "array",
+            "items": {
+                "oneOf": [
+                    {
+                        "type": "object",
+                        "properties": {
+                            "type": {
+                                "type": "string",
+                                "const": "object_store"
+                            },
+                            "url": {
+                                "type": "string",
+                                "format": "uri"
+                            }
+                        },
+                        "required": [
+                            "type",
+                            "url"
+                        ],
+                        "additional_properties": false
```

Comment:
Just found out that this schema has a bug: It is allowing additional properties on every object.

The problem is that the JSON schema field is actually `additionalProperties` in camel case, and not `additional_properties`.

It's also weird that the JSON schema library does not validate the schema of the schemas it loads? lol
Tried finding a "strict" option in the JSON schema but didn't find it anywhere. Might be worth another look.